gameEngine/app.php

```diff
@@ -44,7 +44,7 @@
 if (!empty($_POST['action']) && 'update' == $_POST['action']) {
     $gameHandler->updatePing($_POST['room']);
     $gameHandler->removeInactivePlayer($_POST['room']);
```

Comment:
your session is not available here, yet you are trying to access `$_SESSION['score']` in the method `removeInactivePlayer()`